lib/gauge_qcharge.cu

```diff
@@ -31,7 +31,7 @@ namespace quda
     {
       TuneParam tp = tuneLaunch(*this, getTuning(), getVerbosity());
 
-      std::vector<double> result(3);
+      array<double, 3> result;
```

Comment:
just wondering: Is this std::array or quda::array and since I am wondering would it be better to be more verbose here?

Reply:
This is `quda::array` type, we certainly don't have any `using namespace std;` statements in QUDA.